pages/_app.tsx

```diff
@@ -46,11 +69,12 @@ const InitApp = (): null => {
   useIsTokenPaused()
   useSafeSnapshot()
 
-  // Invalidate caches
-  useContractDelegateInvalidator()
-  useSafeTokenTransferInvalidator()
+  // Only run invalidators when app is open to decrease RPC calls
+  if (isDashboard(pathname)) {
```

Comment:
It actually spams getLogs when run as an app as well. Can we curb them in all situations?

Reply:
Nvm, just realized your TODO above aims to fix that.